packages/ui/src/components/Alert/Alert.stories.tsx

```diff
@@ -40,8 +41,10 @@ const TemplateAlert: ComponentStory<typeof Alert.Root> = ({
             {type === "error" && (
               <Alert.Footer>
                 <p>
-                  <a href="/#">Reach out to our support team</a> if you have any
-                  questions.
+                  <Link external href="/#">
+                    Reach out to our support team
+                  </Link>{" "}
```

Comment:
Stray space

Reply:
Intentional, spaces from the text after.